Makefile

```diff
@@ -116,6 +117,13 @@ ifeq ($(SIR_NO_SYSTEM_LOGGERS),1)
   SIR_CFLAGS += -DSIR_NO_SYSTEM_LOGGERS
 endif
 
+##############################################################################
+# Disable shared library support?
+
+ifeq ($(SIR_NO_SHARED),1)
```

Comment:
I thought the goal was to create a build that doesn't depend on pthreads. How did we get to no shared libraries, and why?

Reply:
Kalray's toolchain (and CompCert-C) doesn't support building shared libraries from the compilers frontend.

Also, the make install would fail due to missing shared libraries if you only built static.  I expect this to mostly be set by the build (like the CompCert section of sirplatform.mk) and not by users directly.

I still want to make a build that doesn't depend on anything from pthreads, but this is mostly unrelated to that since I've been able to (mostly) work around my need for that, for now.